sql/character.anticheat.sql

```diff
@@ -45,6 +45,18 @@ CREATE TABLE IF NOT EXISTS `players_reports_status` (
   PRIMARY KEY (`guid`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
+-- Dumping structure for table characters.lua_cheaters
+DROP TABLE IF EXISTS `lua_cheaters`;
+CREATE TABLE IF NOT EXISTS `lua_cheaters` (
+  `guid` int(11) NOT NULL,
+  `account` int(11) NOT NULL,
```

Comment:
From auth.account table, this should be `int unsigned NOT NULL`

Reply:
What do you need account for btw ? isn't guid unique already ? No other table has `account`

Reply:
Character table has account.  It soo we will know which character specifically used the hack for further evaluation

Reply:
https://github.com/TrinityCore/TrinityCoreCustomChanges/commit/1d1cdda4d4a033b99dde03f0de4606956be0f989